### ISSUE TITLE
Migrate to new OAuth flow

### ISSUE THREAD

**Issue by mcamou** [bcedu/VGrive#130]
**Describe the bug**
When linking a new Google account, you get a warning stating that the app needs to move to the "more secure OAuth flows". There was an email from Google stating that applications that do not use the more secure OAuth flows will be disallowed on October 3.

More information from that email:

What do I need to do?
* Read the [Making Google OAuth interactions safer by using more secure OAuth flows](https://notifications.google.com/g/p/APNL1TiTE5SZT4Y_kA0mS2rChpK9IZ6nAQvzRO4DE4S7xpM8J_Ay2iNjrFog8_WGV3NnFYzT6NIHDQZZFtSeKTFsznmSLcbgeisT4nooOwXR_4jcvyVX49JYT0Ez-gtC5GUWUVnYSm6zs7txWJfP) blog post to learn about the deprecation.
* If your apps still need to use the Google OAuth 2.0 authorization endpoint, follow these migration instructions:
Follow the [Out-of-band (OOB) Migration Guide](https://notifications.google.com/g/p/APNL1TjNwzDGjACNsl1inm8izSOiL6r4m9A4PXPMFi-o-YlkWhGIj6ZBlpvUlMhbgJgS5o7yNvxOTnM0-Wk42ygfVqDTQKfqSUpGAUO5RAKrpuqEzHQ_kXpDZUUUO3bCqkr6rohnpzdwTOfNO5Maz7ooQjTQerw) to migrate your app to an alternative method.
* You may acknowledge the upcoming deadline and suppress a possible user-facing warning message by following instructions in our blog post. All non-compliant authorization requests will be blocked with an invalid_request error when loading Google's OAuth 2.0 authorization endpoint after October 3, 2022.
* If necessary, you may request a one-time [deprecation enforcement extension](https://notifications.google.com/g/p/APNL1TiVWwxF8R3luyZdyy4CJbILgxS8XfwyZPvDFwYo_YDORuN2MUIfVpJWHqkHXS2xbxDxmlUG0VzlUDbmXY8J9pCUqGcIom1uE1S6guM1LysAXvfrY8oaNtKEHYLPUX2w4qoTTzIN4E85oLlB7KT0CUzQWNb4AJEUmS9jq9-4NV8z-Vp7Cie3Lg) for each listed OAuth client ID until January 31, 2023. For clarity, the enforcement for the OOB flow deprecation will be enforced on February 1, 2023, with no exceptions or extensions.
If any of the inactive OAuth clients listed below is no longer needed, remove them from your Google Cloud project by following your client links below.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - VGrive Version v1.6.1 installed from Flatpak

**Comment by eGax** [comment]
Yeah, came across this app, looks very promising, but useless without being able to get it authorized to be used with my Google account. Is there any other way to get it authorized? If not, good luck updating. I'll check back at a later time.

**Comment by tinkicker** [comment]
Maaaaaan! Came here from a lead in a comment about Open Drive for Linux...because it is in the same limbo. Very frustrating.

**Comment by resuna** [comment]
Just installed the app and Google just plain blocks it now.
